### PR TITLE
build/create core module and move structure

### PR DESCRIPTION
**Pull request by amirfathi37**

### CODE REVIEW

File: core/src/main/java/com/gamification/core/CoreApplication.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class CoreApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(CoreApplication.class, args);
+        System.out.println("Hi");
```
Comment (dehghani-reza) [inline core/src/main/java/com/gamification/core/CoreApplication.java]: its seems like application run check please remove it!

File: .gitignore [hunk 2]
```diff
@@ -32,4 +32,4 @@ build/
 ### VS Code ###
 .vscode/
 
-data/
+core/data/
```
Comment (dehghani-reza) [inline .gitignore]: im not sure about this directory change it my ide its in `/data/`

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1,13 +0,0 @@
...
-spring.datasource.driverClassName=org.h2.Driver
-spring.datasource.username=admin
-spring.datasource.password=admin
-spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
-spring.jpa.hibernate.ddl-auto=update
-spring.jpa.show-sql=true
-spring.h2.console.enabled=true
-spring.h2.console.path=/h2-console
-spring.h2.console.settings.trace=false
-spring.h2.console.settings.web-allow-others=false
```
Comment (dehghani-reza) [inline src/main/resources/application.properties]: please remove entire file and directory.
its like empty file in `src` directory

**Review by dehghani-reza — COMMENTED** [review]
A few changes need and ready 4 merge
thank you @amirfathi37

File: src/main/resources/application.properties (hunk 3 above)
Comment (dehghani-reza) [reply src/main/resources/application.properties]: 🔴 **Remove hole directory if it's useless** 

removeplease

File: core/src/main/java/com/gamification/core/CoreApplication.java (hunk 1 above)
Comment (amirfathi37) [reply core/src/main/java/com/gamification/core/CoreApplication.java]: I was for test and I had to delete this statment.

**Review by dehghani-reza — APPROVED** [review]
LGTM 💯

**Pull request merged**